src/k2/cpo/service/CPOService.cpp

```diff
@@ -63,6 +63,10 @@ seastar::future<> CPOService::gracefulStop() {
 seastar::future<> CPOService::start() {
     APIServer& api_server = AppBase().getDist<APIServer>().local();
 
+    if (seastar::smp::count < 2) {
```

Comment:
The idea was that the heartbeat monitor is kept separate so it can be its own service if needed later. I made it configurable though so it can be assigned to the same core if wanted.